README.md

```diff
@@ -1 +1,268 @@
-# node-modules-object-lab
\ No newline at end of file
+# Using Object Module
+
+## Objectives
+
+1. Write a simple non-npm module with module.exports = {}
+1. Import your module from another file
+1. Invoke a method from the module
+
+
+## Introduction
+
+In this lab, you'll master how to create, export and import a object module, i.e., a static module.
```

Comment:
We need to create a "domain" for this lab - the debugger lab was working on accounts at a bank. The more we can connect the labs to work the students can inherently understand, the more motivated they will be. Just having them create modules without a purpose can feel like theyre just doing busy work.